### ISSUE TITLE
[1.16.5] Midas Blessing not working 

### ISSUE THREAD

**Issue by ChromaPIE** [TinyModularThings/UniqueEnchantments#38]
ftf I'm not certain if I misunderstood the description, but one thing for sure is that it's sneak right clicking on the anvil that will consume gold ingots but not sneak lieft clicking.

I successfully stuffed some ingots into a pickaxe and the counter appeared on the tooltips, but mining coal, diamond and emerald ores with it leads to nothing special, 1 drop per ore, nothing got doubled.

Maybe I misunderstood the word **oreGem** in the description but it seems like an Ore Dictionary and afaik in 1.16.5 we use Tags instead of OreDicts.

**Comment by Speiger** [comment]
In 1.16 the version is bugged out.
We kinda  went through all enchantments with a Mod Breaker Tester (Yes he broke every single mod of mine even so i thought they were 100% stable) and we found roughly 15-16 enchantments being broken.

1.12 got the fixes already but then life came away and the other two versions had to be delayed... Yes should be addressed next weekend hopefully.

TL:DR: We know, we had a giant testing spree, we already planed a patch, 1.12 managed to get done but then I ran out of time...
ETA is next weekend.
